OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -62,6 +58,12 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    private void handleExtras() {
```

Comment:
`parseExtras()` or `getExtras()` would be better names in my opinion, what do you think?

Reply:
`parseExtras()` sounds good to me.